source/windows/condition_variable.c

```diff
@@ -0,0 +1,56 @@
+/*
+* Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License").
+* You may not use this file except in compliance with the License.
+* A copy of the License is located at
+*
+*  http://aws.amazon.com/apache2.0
+*
+* or in the "license" file accompanying this file. This file is distributed
+* on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+* express or implied. See the License for the specific language governing
+* permissions and limitations under the License.
+*/
+
+#include <aws/common/condition_variable.h>
+#include <aws/common/mutex.h>
+
+int aws_condition_variable_init(struct aws_condition_variable *condition_variable) {
+    InitializeConditionVariable(&condition_variable->condition_handle);
+    return AWS_OP_SUCCESS;
+}
+
+void aws_condition_variable_clean_up(struct aws_condition_variable *condition_variable) {
+    (void *)&condition_variable;
```

Comment:
I'd remove this line and remove the name of the parameter to avoid unused variable warnings.

Reply:
that's not allowed in C for function definitions (only declarations).